geode-core/src/main/java/org/apache/geode/internal/cache/Oplog.java

```diff
@@ -937,17 +937,11 @@ void initAfterRecovery(boolean offline) {
         // this.crf.raf.seek(this.crf.currSize);
       } else if (!offline) {
         // drf exists but crf has been deleted (because it was empty).
+        // I don't think the drf needs to be opened. It is only used during
+        // recovery.
+        // At some point the compacter my identify that it can be deleted.
```

Comment:
change "my" to "may"

Reply:
Thanks @dschneider-pivotal!. I'll fix the type and create another ticket for the long term solution.